cluster-provision/gocli/cmd/run.go

```diff
@@ -194,6 +194,21 @@ func run(cmd *cobra.Command, args []string) (err error) {
 		}
 	}
 
+	// Mount /lib/modules at dnsmasq if it's there since sometimes
+	// some kernel modules may be mounted
+	dnsmasqMounts := []mount.Mount{}
+	_, err = os.Stat("/lib/modules")
```

Comment:
you can combine to one liner
if .... ; err == nil {

Reply:
I know I can but I am not going to do it, every line should have as less responsibility as possible.

Reply:
i honestly dont understand why, in this specific situation when we just need both for one thing, but nvm.

Reply:
Error handling is one thing and opening a file/dir is other, so they belong to different lines as specified at effective golang, I am going to resolve this.